sdk/apimanagement/Azure.ResourceManager.ApiManagement/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Release History
 
+## 2.0.0 (Unreleased)
```

Comment:
- Please remove `1.3.0-beta.2`. We should not have two `Unreleased` versions.
- If you want to publish a release, replace `Unreleased` with a specific date like `2024-12-10`